pycommon/reporter_config/actions/File.py

```diff
@@ -17,18 +18,20 @@ class FileAction(Action):
         * STDOUT: message is written to STDOUT pipe
     """
     def __init__(self, action):
-        self.actionId = action["id"]
-        self.actionType = "file"
-        self.path = action["file"]["path"]
-        if "dir" in action["file"]:
-            self.dir = action["file"]["dir"]
-        else:
-            try:
-                # path already exists, check if it is a directory
-                s = os.stat(self.path)
-                self.dir = S_ISDIR(s.st_mode)
-            except:
-                self.dir = False
+        super(type(self), self).__init__(actionId = action["id"], actionType = "file")
+
+        a = action["file"]
+
+        self.path = a["path"] if "path" in a else None
+        if not self.path:
+            raise Exception("File action requires `path` argument.")
+
+        try:
+            # path already exists, check if it is a directory
+            s = os.stat(self.path)
+            self.dir = S_ISDIR(s.st_mode)
+        except:
+            self.dir = False
 
         if self.path == '-':
             self.fileHandle = sys.stdout
```

Comment:
(comment to line 38)
Why `dir=True` when input is stdout? It's confusing and I don't see any reason for it.